src/exchange/group/delegation/group-delegation.html

```diff
@@ -23,6 +23,16 @@
                 <div class="form-group">
                     <label class="sr-only" for="delegationSearch"
                            data-translate="common_search"></label>
+                    <oui-spinner data-size="s" data-ng-if="ctrl.loadingDomains"></oui-spinner>
+                    <div class="oui-select oui-select_inline" data-ng-if="!ctrl.loadingDomains">
```

Comment:
Wouldn't a `oui-select` work here ?

Reply:
Looks like it's not possible or too complex to get it done. I need to add a custom option to select all domains. On selecting it, all domains are shown.

Reply:
I replaced select with oui-select